From 0250abe3e304b348fa7ad719f035b0f4ed910365 Mon Sep 17 00:00:00 2001
From: Axel Svensson <mail@axelsvensson.com>
Date: Fri, 5 Sep 2025 08:57:08 +0200
Subject: [PATCH 2/3] Help text formatting

- Reflow to output max 79 columns.
- Add a `=` to `--ignore-pid` for consistency.
---
 fatrace.c | 29 +++++++++++++++++------------
 1 file changed, 17 insertions(+), 12 deletions(-)

diff --git a/fatrace.c b/fatrace.c
index e683535..836c431 100644
--- a/fatrace.c
+++ b/fatrace.c
@@ -647,18 +647,23 @@ help (void)
     puts ("Usage: fatrace [options...] \n"
 "\n"
 "Options:\n"
-"  -c, --current-mount\t\tOnly record events on partition/mount of current directory.\n"
-"  -o FILE, --output=FILE\tWrite events to a file instead of standard output.\n"
-"  -s SECONDS, --seconds=SECONDS\tStop after the given number of seconds.\n"
-"  -t, --timestamp\t\tAdd timestamp to events. Give twice for seconds since the epoch.\n"
-"  -u, --user\t\t\tAdd user ID and group ID to events.\n"
-"  -p PID, --ignore-pid PID\tIgnore events for this process ID. Can be specified multiple times.\n"
-"  -f TYPES, --filter=TYPES\tShow only the given event types; choose from C, R, O, W, +, D, < or >, e. g. --filter=OC.\n"
-"  -C COMM, --command=COMM\tShow only events for this command.\n"
-"  -j, --json\t\t\tWrite events in JSONL format.\n"
-"  -P, --parents\t\t\tInclude information about all parent processes.\n"
-"  -e, --exe\t\t\tAdd executable path to events.\n"
-"  -h, --help\t\t\tShow help.");
+"  -c, --current-mount           Only record events on partition/mount of\n"
+"                                current directory.\n"
+"  -o FILE, --output=FILE        Write events to a file instead of standard\n"
+"                                output.\n"
+"  -s SECONDS, --seconds=SECONDS Stop after the given number of seconds.\n"
+"  -t, --timestamp               Add timestamp to events. Give twice for seconds\n"
+"                                since the epoch.\n"
+"  -u, --user                    Add user ID and group ID to events.\n"
+"  -p PID, --ignore-pid=PID      Ignore events for this process ID. Can be\n"
+"                                specified multiple times.\n"
+"  -f TYPES, --filter=TYPES      Show only the given event types; choose from C,\n"
+"                                R, O, W, +, D, < or >, e. g. --filter=OC.\n"
+"  -C COMM, --command=COMM       Show only events for this command.\n"
+"  -j, --json                    Write events in JSONL format.\n"
+"  -P, --parents                 Include information about all parent processes.\n"
+"  -e, --exe                     Add executable path to events.\n"
+"  -h, --help                    Show help.");
 }
 
 /**
-- 
2.46.3

